packages/client/hmi-client/src/components/responsive-matrix/cell-line-graph.vue

```diff
@@ -225,20 +227,50 @@ export default {
 				.attr('viewBox', `0 0 ${width} ${height}`)
 				.style('background', 'white');
```

Comment:
you need to import [Uncharted Design Tokens](https://github.com/unchartedsoftware/design-tokens/blob/main/build/js/es6/uncharted-tokens.js)

```ts
import { UnColorWhite } from '@node_modules/@uncharted.software/design-tokens/build/js/es6/uncharted-tokens.js';
```

```suggestion
				.style('background', UnColorWhite);
```